src/app/models/Site.ts

```diff
@@ -82,3 +82,12 @@ export class Site implements SiteInterface {
     });
   }
 }
+
+export const mockSite = new Site({
+  id: 1,
+  name: "name",
+  description: "description",
+  creatorId: 1,
+  projectIds: new Set([0]),
+  imageUrl: "/assets/images/site/site_span4.png"
+});
```

Comment:
Creating a mock site may be a bad practice for the unit tests which implement it